README.md

```diff
@@ -46,3 +46,11 @@ jobs:
         with:
           args: Sent from GitHub Actions with secrets from SecretHub 🔑
 ```
+
+#### Credential
```

Comment:
Could also be an `h2` because it applies to SecretHub (Actions) in general.

```suggestion
## Credential
```

Reply:
Shall we change the readme to allow for multiple actions when we get there? For now, I think it's helpful if the credential section refers to the code snippet in the usage. It currently does so twice. I'm not sure whether that makes sense if it's in a separate section?

Reply:
Reason I brought it up in the first place is because it looked very tiny and 'off' with the rest of the page.